rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -22,8 +22,10 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.8
 Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0
 Requires: %{?scl_prefix}rubygem(dynflow) >= 0.8.14
 Requires: %{?scl_prefix}rubygem(dynflow) < 0.9.0
-Requires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 1.0
-Requires: %{?scl_prefix}rubygem(foreman_ansible_core) < 2.0
+Requires: %{?scl_prefix}rubygem(foreman_ansible_core) >= 2.0
```

Comment:
It's OK if it's not 2.0.1 - the difference between `foreman_ansible_core` 2.0.0 and 2.0.1 is that 2.0.1 has a dependency on `foreman_Remote_execution_core`, which I forgot to add on 2.0.0.

Since this plugin already depends on `foreman_remote_execution`, which depends on `foreman_remote_execution_core`, the dependency is surely fulfilled.

In addition to that there are no packages for foreman_ansible_core 2.0.0